src/main/java/org/wildfly/security/_private/ElytronMessages.java

```diff
@@ -590,8 +590,8 @@
     @Message(id = 1145, value = "Security realm [%s] must implement [%s]")
     IllegalArgumentException realmCacheUnexpectedType(SecurityRealm realm, Class<? extends CacheableSecurityRealm> expectedType);
 
-    @Message(id = 1146, value = "Ldap realm failed to register notification listener")
-    IllegalStateException ldapRealmFailedRegisterListener(@Cause  Throwable cause);
+    //@Message(id = 1146, value = "Ldap realm failed to register notification listener")
+    //IllegalStateException ldapRealmFailedRegisterListener(@Cause  Throwable cause);
```

Comment:
I think you can remove the removed message completely and put the new message on its place - no need to keep it commented out here.